action/receipt.go

```diff
@@ -95,7 +95,7 @@ func (log *Log) ConvertToLogPb() *iotextypes.Log {
 	l.ContractAddress = log.Address
 	l.Topics = [][]byte{}
 	for _, topic := range log.Topics {
-		if log.PreAleutian {
+		if log.NotFixTopicCopyBug {
 			l.Topics = append(l.Topics, topic[:])
 		} else {
 			data := make([]byte, len(topic))
```

Comment:
can you improve the fix?

for i, topic := range log.Topics {
    else {
         l.Topics = append(l.Topics, log.Topics[i][:])

so can avoid an extra make() and copy()

Reply:
not related to this pr